### PR TITLE
Restyle W3W FE

### LINKED ISSUE #3: Restyle
Restyle FE

### LINKED ISSUE #3: Pull FAQs from the CMS
https://web3wines.org/faqs

### PR DESCRIPTION
**Pull request by Maiha192**
Fixes #3

Home:
Screenshot 2023-10-11 at 11 03 14 am

Footer:
Screenshot 2023-10-11 at 11 03 22 am

About us:
Screenshot 2023-10-11 at 11 03 30 am

Our Team:
Screenshot 2023-10-11 at 11 03 37 am

Value chain:
Screenshot 2023-10-11 at 11 03 45 am

Blockchain application:
Screenshot 2023-10-11 at 11 03 53 am

Membership:
Screenshot 2023-10-11 at 11 04 03 am

News & Updates:
Screenshot 2023-10-11 at 11 04 11 am

FAQs:
Screenshot 2023-10-11 at 11 04 19 am

Contact us:
Screenshot 2023-10-11 at 11 04 27 am
Screenshot 2023-10-11 at 11 04 36 am

### CODE REVIEW

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+export function Footer() {
+    return (
+        <footer className="relative w-full bg-navbar py-3">
+            <div className="mx-auto w-full px-5 text-center">
+                <div className="gap-4">
+                    <div className="text-white font-bold">WEB3 WINES</div>
+                    <div>
+                        <p>Address: 1 Eagle St., Brisbane QLD 4000</p>
+                        <p>Email: info@web3wines.org</p>
+                        <p>Phone: (04)2345678</p>
```
Comment (bitcoinbrisbane) [inline src/components/Footer.tsx]: Probably remove this line or add a real phone number.

File: src/components/NavBar.tsx
Comment (bitcoinbrisbane) [inline src/components/NavBar.tsx:1]: Need to lint this file.  Some lines dont have ; or double quotes.

File: src/pages/FAQs.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React, { useLayoutEffect } from "react";
+import { PageLayout } from "../components";
+
+const faqs = [
+    { id: 1, question: 'What is NFT?', answer: 'An NFT (non-fungible token) is a unique digital identifier that is recorded on a blockchain, and is used to certify ownership and authenticity. It cannot be copied, substituted, or subdivided. ' +
+            'The ownership of an NFT is recorded in the blockchain and can be transferred by the owner, allowing NFTs to be sold and traded.'},
+    { id: 2, question: 'What are some examples of using blockchain in wine industry?', answer: 'BitWine, Winechain, Cult Wines'},
+
+
+
```
Comment (bitcoinbrisbane) [inline src/pages/FAQs.tsx]: lets remove this white space.

**Review by bitcoinbrisbane — CHANGES_REQUESTED** [review]
Few minor things and linting :)

File: src/components/Footer.tsx (hunk 1 above)
Comment (Maiha192) [reply src/components/Footer.tsx]: issue fixed

File: src/components/NavBar.tsx
Comment (Maiha192) [reply src/components/NavBar.tsx:1]: issue fixed

File: src/pages/FAQs.tsx (hunk 2 above)
Comment (Maiha192) [reply src/pages/FAQs.tsx]: issue fixed

**Review by bitcoinbrisbane — APPROVED** [review]

**Pull request merged**